CODEOWNERS

```diff
@@ -192,6 +192,8 @@ extensions/filters/http/oauth2 @derekargueta @mattklein123
 /*/extensions/filters/http/rate_limit_quota @tyxia @yanavlasov
 # HTTP Bandwidth Limit
 /*/extensions/filters/http/bandwidth_limit @nitgoy @mattklein123 @yanavlasov @tonya11en
+# HTTP Basic Auth
+/*/extensions/filters/http/basic_auth @zhaohuabing
```

Comment:
Please add sponsor (me) and initial reviewer (still me) to the owner list, then format checker wont complain. :)